index.js

```diff
@@ -1530,12 +1534,23 @@ class Command extends EventEmitter {
         return passthru;
       };
     }
+    globalEventEmitter.emit(this._helpHeader, this);
+    this.emit(this._helpHeader, this);
     const cbOutput = cb(this.helpInformation());
```

Comment:
In theory we should try and include or make available cb in the processing of the listener too, but that callback is a pattern I would like to deprecate. Instead, the processing of the callback can shift into the non-override body block since no need to call if not being displayed.